src/components/CompositeBar/MultipleTooltip/MultipleTooltip.scss

```diff
@@ -0,0 +1,76 @@
+@use '../../variables';
+
+$block: '.#{variables.$ns}multiple-tooltip';
+
+.yc-root_theme_dark #{$block},
+.yc-root_theme_dark-hc #{$block} {
+    --item-bg-color: #424147; // TODO: color variable will appear in uikit 5
```

Comment:
May we use more specific names e.g.`composite-bar-item-bg-color`?
I'm afraid of intersections with outside components